docs/arcaflow/workflows/flow-control.md

```diff
@@ -285,6 +338,55 @@ outputs:
     plugin_output: $.steps.my_step.outputs.success
 ```
 
+### Oneof methods with step flow control
+
+It is also possible to use oneof conditions as part of the `wait_for` flow control. An
+explicit relationshp between steps in a workflow in this case becomes a oneof condition
+for either the requested step output or otherwise the disabled state of the step.
```

Comment:
The "otherwise" feels unnecessary here. This also implies explaining the implicit assumption that you're using "oneof" here because you're either running or disabling a step.